src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -164,7 +165,10 @@ export class ExtensionEditor extends BaseEditor {
 	private navbar: NavBar;
 	private content: HTMLElement;
 	private recommendation: HTMLElement;
+	private recommendationText: any;
+	private ignoreActionbar: ActionBar;
 	private header: HTMLElement;
+	private recentlyIgnored: string[] = [];
```

Comment:
I believe we dont need to maintain the `recentlyIgnored` list.
As soon as a recommendation is ignored, we update the text in the callback. That should be enough.